front/src/applications/stdcm/components/StdcmForm/StdcmDestination.tsx

```diff
@@ -39,8 +39,14 @@ const StdcmDestination = ({
         ? extractDateAndTimefromISO(destination.arrival)
         : undefined,
       destinationToleranceValues: {
-        arrivalToleranceBefore: destination?.arrivalToleranceBefore || DEFAULT_TOLERANCE,
-        arrivalToleranceAfter: destination?.arrivalToleranceAfter || DEFAULT_TOLERANCE,
+        arrivalToleranceBefore:
+          destination?.arrivalToleranceBefore === undefined
+            ? DEFAULT_TOLERANCE
+            : destination.arrivalToleranceBefore,
+        arrivalToleranceAfter:
+          destination?.arrivalToleranceAfter === undefined
+            ? DEFAULT_TOLERANCE
+            : destination.arrivalToleranceAfter,
```

Comment:
Same here, the behaviour is not the same ?

(nit: it could be moved to a new commit)